src/pages/login.tsx

```diff
@@ -33,9 +33,16 @@ export default function LoginPage() {
     const onSubmit: SubmitHandler<FormFields> = async data => {
         try {
             const { meta: responseData } = await dispatch(loginUser(data))
+            const user = localStorage.getItem('user')
+            const userData = JSON.parse(user || '{}')
             if (responseData.requestStatus === 'fulfilled') {
-                navigate('/dashboard')
-                toast.success('Successfully logged in!')
+                if (userData.roles === 'Admin') {
+                    navigate('/workspace')
```

Comment:
use naming that doesn't confuse people. the word workspace refers to something else here